GameFiles/GameInterface/_integration_base.gsc

```diff
@@ -22,6 +20,8 @@ Setup()
     level.commonFunctions              = spawnstruct();
     level.commonFunctions.setDvar      = "SetDvarIfUninitialized";
     level.commonFunctions.isBot        = "IsBot";
+    level.commonFunctions.getXuid      = "GetXuid";
+    level.commonFunctions.GetPlayerFromClientNum = "GetPlayerFromClientNum";
```

Comment:
nit: use camelCase for functionName
`level.commonFunctions.getPlayerFromClientNum`